### PR TITLE
Add write gtf feature - Issue #21

### PR DESCRIPTION
**Pull request by Benoitdw**
Fixe #21

### CODE REVIEW

File: tests/test_write_gtf.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from .data import data_path
+from polars import DataFrame
+
+REFSEQ_GTF_PATH = data_path("refseq.ucsc.small.gtf")
+
+
+def test_write_gtf(tmp_path):
+    gtf_dict = read_gtf(REFSEQ_GTF_PATH)
+    write_gtf(gtf_dict, tmp_path/"dummy_gtf.gtf")
+    assert  isinstance(read_gtf(str(tmp_path/"dummy_gtf.gtf")), DataFrame)
```
Comment (gamazeps) [inline tests/test_write_gtf.py:11]: Check for equality of this gtf and the original one too ?

Comment (Benoitdw) [reply tests/test_write_gtf.py]: Is it better now ?

Comment (gamazeps) [reply tests/test_write_gtf.py]: Yup 👍

**Comment by iskandr** [comment]
Thanks @Benoitdw — sorry this sat so long! I've adopted your `write_gtf` and carried it forward in #72, which is now open against current `master`. Your authorship is preserved there via a `Co-authored-by` trailer.

I made a few fixes on top of your version so it could land:
- **Python 3.9 compat** — the `str | Path` union syntax needs 3.10+, but the project targets `>=3.9`, so I switched to `typing.Union`/`Optional`.
- **No silent data loss** — the `and (row[field])` filter dropped attributes whose value was falsy (`0`, `""`, `False`); attributes are now omitted only when the value is `None`, so those survive a round trip.
- Missing fixed-column values now serialize as `.`, the attribute field ends with a trailing `;` (canonical GTF), and `expand_attribute_column=False` DataFrames round-trip too.
- Added round-trip tests for RefSeq + Ensembl (expanded and unexpanded), pandas input, and the edge cases above.

Closing this in favor of #72 — really appreciate the original contribution, which is what got the feature moving. 🙏
